src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.ForEachAsync.cs

```diff
@@ -183,9 +183,9 @@ private static Task ForEachAsync<TSource>(IEnumerable<TSource> source, int dop,
             }
         }
 
-        /// <summary>Executes a for each operation on an <see cref="System.Collections.Generic.IEnumerable{TSource}"/> in which iterations may run in parallel.</summary>
+        /// <summary>Executes a for each operation on an <see cref="System.Collections.Generic.IAsyncEnumerable{TSource}"/> in which iterations may run in parallel.</summary>
```

Comment:
There are other occurrences of these elsewhere in the file.  Those should be changed, too.  Thanks.

Reply:
Oops, no, those are actually enumerables.  Ignore that.